content/actions/creating-actions/publishing-actions-in-github-marketplace.md

```diff
@@ -64,6 +64,15 @@ To remove a published action from {% data variables.product.prodname_marketplace
 1. Select **Publish this action to the {% data variables.product.prodname_marketplace %}** to remove the check from the box.
 1. Click **Update release** at the bottom of the page.
 
+## Transferring action repositories
+
+Action repositories can be moved by exporting or cloning the original repository, making a new repository on the target platform, and then pushing the code to the new repository.
+
+- A repo can be transferred and actions will continue to work because there is a redirect in place (users of the action won't have to update their workflows)
+- An action in the marketplace is linked to a repository by ID, allowing for new releases to be published under the same marketplace listing from the "new" repo
+- If an action repository is deleted, so is the marketplace listing. The unique `name` identifier becomes available again.
+- The `verified` status seen on GitHub org profiles is different from the marketplace `verified` badge. By transferring an action repo, the owner changes and the marketplace badge will be lost unless the new owner is a verified publisher too.
```

Comment:
```suggestion

{% note %}

**Note:** The "Verified" badge seen on an organization's {% data variables.product.prodname_dotcom %} profile is different from the verified creator badge on {% data variables.product.prodname_marketplace %}. If you transfer an action repository, the {% data variables.product.prodname_marketplace %} listing will lose the verified creator badge unless the new owner is also a verified creator.

{% endnote %}
```

Since this is an important callout for someone thinking about transferring an action repository, I've put this in a note block so it will appear as a bright blue callout on the page (for reference, see "[Publishing an action](https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace#publishing-an-action)" in this article). I've also made some style guide updates to use variables and active voice.